analyses/molecular-subtyping-HGG/results/HGG_cleaned_mutation.tsv

```diff
@@ -63,11 +63,12 @@ PT_DMAF1J4A	A18552	BS_JCWD4DMA	Yes	No	No	No	No	No	H3F3A, TP53	No R132 or R172	No
 PT_EN2RN5Y1	A17997	BS_328C4BW5	Yes	No	No	No	No	No	ATRX, H3F3A	No R132 or R172	None	No V600E
 PT_FYPWZBH0	7316-1085	BS_ZCY4ECNV	Yes	No	No	No	No	No	H3F3A, TP53	No R132 or R172	None	No V600E
 PT_G16VK7FR	A19649	BS_2HDEZFTW	No	Yes	No	No	No	No	HIST1H3B, TP53	No R132 or R172	None	No V600E
-PT_HGM20MW7	7316-3221	BS_FK3B5SDH	Yes	No	No	No	No	No	BRAF, H3F3A	No R132 or R172	None	p.V600E
-PT_HGM20MW7	A09446	BS_52ETE050	Yes	No	No	No	No	No	BRAF, H3F3A, TP53	No R132 or R172	None	p.V600E
+PT_HGM20MW7	7316-3221	BS_FK3B5SDH	Yes	No	No	No	No	No	BRAF, H3F3A	No R132 or R172	None	No V600E
+PT_HGM20MW7	A09446	BS_52ETE050	Yes	No	No	No	No	No	BRAF, H3F3A, TP53	No R132 or R172	None	No V600E
```

Comment:
Noting that there are a number of changes here in the BRAF_V600E column: There are now NO samples with p.V600E. Is this expected?

Reply:
Yes, we removed it from HGG subtyping with #853

Reply:
Actually, we should remove that column from this table

Reply:
Okay, that is a better than I had feared. I was worried something new had gone wrong with the SNV consensus.

Should we also remove (or comment out) the [`09-HGG-with-braf-clustering.Rmd`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-HGG/09-HGG-with-braf-clustering.Rmd) notebook then as well?

Reply:
Oh shoot. That notebook is actually being retained because it led to us investigate and change a `pathology_diagnosis`, and as I look at the code, it is pulling in this information from the above mutation file: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/fbdb9a004fa667f431af0c983d0c4f71f4149939/analyses/molecular-subtyping-HGG/09-HGG-with-braf-clustering.Rmd#L94
per [this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/856#issuecomment-740006900).

So now, I do not think this is expected.

Reply:
Thanks for catching this! 
I checked the data/pbta-snv-consensus-mutation.maf.tsv.gz in v19 and it seems that the BRAF calls are at p.V640E instead of p.V600E 👀  that's why it is not captured in this module. 

Chromosome | Start_Position | End_Position | Strand | Variant_Classification | Tumor_Sample_Barcode | Hugo_Symbol | HGVSp_Short | sample_id | Kids_First_Participant_ID
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
chr7 | 140753336 | 140753336 | + | Missense_Mutation | BS_CZRA594T | BRAF | p.V640E | 7316-85 | PT_3AWKWXEV
chr7 | 140753336 | 140753336 | + | Missense_Mutation | BS_ERAWW3H7 | BRAF | p.V640E | 7316-85 | PT_3AWKWXEV
chr7 | 140753336 | 140753336 | + | Missense_Mutation | BS_E42SSQGJ | BRAF | p.V640E | 7316-85 | PT_3AWKWXEV
chr7 | 140753336 | 140753336 | + | Missense_Mutation | BS_KAD49R68 | BRAF | p.V640E | 7316-3521 | PT_63F63QY0
chr7 | 140753336 | 140753336 | + | Missense_Mutation | BS_FK3B5SDH | BRAF | p.V640E | 7316-3221 | PT_HGM20MW7
chr7 | 140753336 | 140753336 | + | Missense_Mutation | BS_52ETE050 | BRAF | p.V640E | A09446 | PT_HGM20MW7

Reply:
copying in @migbro here because this makes it seem like the calls in strelka, mutect, and lancet would be called this way- unless something odd happened in consensus. i have not yet downloaded the files, but will do so now..

Reply:
Is what is happening here that the gencode annotation version used for the MAF file generation is different? If so, we may have some other updates to do. This could actually be a pretty deep problem, unfortunately, as all of the RNAseq data is presumably quantified with v27, as we were using before...

Reply:
[This](https://github.com/kids-first/kf-mskcc-vcf2maf/blob/main/data/isoform_overrides_uniprot) is the reference file for the above parameter.

Reply:
Update on this. @migbro did a test without the uniprot override param and we recovered the old transcript and BRAF V600E change, so that was the culprit. He will rerun TCGA and PBTA vcf2maf overnight tonight and we can add new files to s3 tomorrow. We can rerun consensus @jashapiro since you'll be out, and if looks good, regenerate for CI and have those ready for uploading for Monday. Cc @kgaonkar6

Reply:
Sounds good. Just note you will need to use the updated script in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1033 because of the change in the MAF files. If you run without updating the comparison notebooks (the defaults with `run_caller_consensus_analysis-pbta.sh` and `run_caller_consensus_analysis-tcga.sh`) the scripts will finish much more quickly.